ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py

```diff
@@ -727,3 +726,42 @@ def test_paddle_tanh(
         frontend=frontend,
         on_device=on_device,
     )
+
+
+# concat
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="paddle.to_tensor",
+    method_name="concat",
+    dtype_x_axis=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        num_arrays=st.integers(min_value=1, max_value=4),
+        min_num_dims=1,
+        shared_dtype=True,
+    ),
+)
+def test_paddle_concat(
+    dtype_x_axis,
+    frontend_method_data,
+    init_flags,
+    method_flags,
+    frontend,
+    on_device,
+):
+    input_dtype, x, axis = dtype_x_axis
+    helpers.test_frontend_method(
+        init_input_dtypes=input_dtype,
+        init_flags=init_flags,
+        init_all_as_kwargs_np={
```

Comment:
Please correct the tests according to the functions you'll modify.

There are some great docs on [testing](https://unify.ai/docs/ivy/docs/ivy_tests.test_ivy.helpers.html) and [YouTube](https://www.youtube.com/watch?v=iS7QFsQa9bI) videos, which will clear your concepts. Just thought to share if you need any heads-up.